printer_mods/README.md

```diff
@@ -229,6 +229,7 @@ like so:
 | rsheng118 | [BTT octopus DIN vertical mount](./rsheng118/BTT_octopus_DIN_vertical_mount) | DIN vertical mount for Big Tech Tree Octopus board | :x: | :x: | :heavy_check_mark: | :x: |
 | See1807 | [TopHat Exhaust and LED Light](./See1807/Voron_V0_Exhaust_and_LED_Mod) | A mount for 10mm LED strips and Exhaust for Voron V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | sse018 | [Octopus DIN Rail bracket](./sse018/Octopus_DIN_Rail_bracket) | A Din rail mount for BTT Octopus board | :x: | :x: |:heavy_check_mark: | :x: |
+| Slidr | [PanzerBalls 2.4](./Slidr/PanzerBalls2.4) | Squash Ball Feet for V2.4 | :x: | :x: | :heavy_check_mark: |:x:|
```

Comment:
Please add an `:x:` (or question mark or checkmark) in the Trident column.

Please have your user name on only the first line, and the rest of your mods would have a blank for the name.  So your two mods would look like

```
| Slidr | [PanzerBalls 2.4](./Slidr/PanzerBalls2.4) | Squash Ball Feet for V2.4 | :x: | :x: | :heavy_check_mark: | :x: | :grey_question: |
| | [PanzerChain 2.4](./Slidr/PanzerChain2.4) | Energychain for V2.4 | :x: | :x: | :heavy_check_mark: | :x: | :x: |
```

it would display in the UI something like
<img width="1048" alt="Screen Shot 2021-10-02 at 16 04 31" src="https://user-images.githubusercontent.com/69389593/135730428-d08a74ff-8285-4f3b-8205-381b8f48927e.png">